service/src/main/java/greencity/security/service/OwnSecurityServiceImpl.java

```diff
@@ -184,7 +196,10 @@ public SuccessSignUpDto signUpEmployee(EmployeeSignUpDto employeeSignUpDto, Stri
         String password = generatePassword();
         employeeSignUpDto.setPassword(password);
         OwnSignUpDto dto = modelMapper.map(employeeSignUpDto, OwnSignUpDto.class);
+
         User employee = createNewRegisteredUser(dto, jwtTool.generateTokenKey(), language);
+        employee.setRole(Role.ROLE_UBS_EMPLOYEE);
+        setUsersFieldsEmployee(dto, employee);
```

Comment:
This method `setUsersFieldsEmployee` contains the same logic as `setUsersFields` method and also next logic:

```
RestorePasswordEmail restorePasswordEmail = createRestorePasswordEmail(employee, jwtTool.generateTokenKey());
employee.setRestorePasswordEmail(restorePasswordEmail);
employee.setUuid(UUID.randomUUID().toString());
```

Why do we need to call `setUsersFields` method too?